src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let previous = initialValue;
```

Comment:
```suggestion
    let accumulationvalue = initialValue;
```

Reply:
Not critical, but better for understanding

Reply:
> Not critical, but better for understanding

Corrected !